site/spi/Makefile-bits-arnold

```diff
@@ -108,6 +111,8 @@ ifeq (${SP_OS}, rhel7)
     MY_CMAKE_FLAGS += \
 	-DOCIO_PATH=${OCIO_PATH} \
 	-DFIELD3D_HOME=${FIELD3D_HOME} \
+    -DOPENVDB_LOCATION=${OPENVDB_LOCATION} \
+    -DTBB_LOCATION=${TBB_LOCATION} \
```

Comment:
I think we should expose a TBB_LOCATION cached variable (and a way to set it via the makefile wrapper?). And maybe a USE_TBB option?